APIScrape/README.md

```diff
@@ -0,0 +1,70 @@
+TheCatApi // Developer Experience
```

Comment:
This document should have some titles - I'll merge it without, but for future reference use:
`# Title` syntax and add more `##`s for smaller titles (in markdown).